node/building_blocks.re

```diff
@@ -58,6 +58,22 @@ let is_current_producer = (state, ~key) => {
 // TODO: bad naming
 // TODO: check if block must have published a new snapshot
 let is_signable = (state, block) => {
+  open Protocol.Operation.Side_chain;
+  let trusted_validators = state.Node.trusted_validators;
+  let rec contains_only_trusted_add_validator_op = result_so_far =>
+    fun
+    | [] => result_so_far
+    | [h, ...t] =>
+      switch (h.kind) {
+      | Add_validator(validator) =>
```

Comment:
Remove_validator should also be validated, otherwise someone could remove all the validators of the chain

Reply:
Addressed in https://github.com/marigold-dev/sidechain/pull/182